docs/design/coreclr/profiling/Profiler Loading.md

```diff
@@ -1,23 +1,39 @@
+**Note:**
```

Comment:
This should be also documented in https://learn.microsoft.com/en-us/dotnet/core/runtime-config/debugging-profiling . The source for the official documentation lives at https://github.com/dotnet/docs/blob/main/docs/core/runtime-config/debugging-profiling.md 

It may be best to delete the redundant documentation here and merge this doc into the official docs.

Reply:
Yes, I think it would be the best.